### PR TITLE
Rename type changes

### PR DESCRIPTION
**Pull request by mark-todd**

### CODE REVIEW

File: mat_dp_pipeline/presentation/application.py [hunk 1]
```diff
@@ -72,7 +71,7 @@ def __init__(self, outputs: PipelineOutput, path_leaf_split: list[str]):
         self.dash_app = Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])
         self.outputs = outputs
         self.indicators = sorted(self.outputs.indicators)
-        self.paths = typing.cast(list[Path], sorted(self.outputs.keys(Path)))
+        self.paths: list[Path] = sorted(self.outputs.by_path.keys())
```
Comment (mkcinu) [inline mat_dp_pipeline/presentation/application.py:74]: `: list[Path]` isn't needed.

File: mat_dp_pipeline/sdf/standard_data_format.py [hunk 2]
```diff
@@ -253,12 +255,15 @@ def dfs(root: Path) -> StandardDataFormat | None:
 
             for inten in filter(lambda df: not df.empty, all_intensities):
```
Comment (mkcinu) [inline mat_dp_pipeline/sdf/standard_data_format.py:256]: Now can be `intensities` :)

Comment (mark-todd) [reply mat_dp_pipeline/sdf/standard_data_format.py:256]: haha lol I guess so

Comment (mark-todd) [reply mat_dp_pipeline/sdf/standard_data_format.py:256]: Will see when we've reviewed the todo

**Pull request merged**